apps/core/app/[locale]/(default)/cart/_queries/get-shipping-countries.ts

```diff
@@ -3,6 +3,7 @@ import { getShippingZones } from '~/client/management/get-shipping-zones';
 
```

Comment:
I moved these files because they are not really `actions`, but mostly functions that call queries, not sure if this is the best approach.